engine-api/src/schema/eth/block.rs

```diff
@@ -0,0 +1,16 @@
+pub use alloy::eips::BlockNumberOrTag;
+
+use {
+    alloy::rpc,
+    moved::types::state::BlockResponse,
+    serde::{Deserialize, Serialize},
+};
+
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+pub struct GetBlockResponse(rpc::types::Block<rpc::types::Transaction>);
```

Comment:
```suggestion
pub struct GetBlockResponse(pub rpc::types::Block<rpc::types::Transaction>);
```

Maybe want to make the inner value `pub` in case we want to use it somewhere in the code.